### DISCUSSION TITLE
add network storage as option in hassio version

### DISCUSSION THREAD

**Discussion by DAVIZINH0** [blakeblackshear/frigate#6901] [Ideas]
Hello,
Now with last version of hassio (2023 june) we can mount in home assistant a network storage.
Its posible to config in frigate addon that use one network storage of hassio?

I think this will be very interesting

**Comment by NickM-27** [comment]
This is already supported, I have a PR up to update the docs but you simply need to create the mount in HA with the name `frigate`

**Reply by DAVIZINH0** [reply]
wow!!! if i create one with the name "frigate" it works now???? so simple!!!  or in a future update?
im exciting now! jeje

**Reply by NickM-27** [reply]
yes it works, no changes are needed from frigate side

**Reply by DAVIZINH0** [reply]
I try but not works. im sure its my fault, its to late in spain. Tomorrow I will try again
Only one question, what kind of storage  i must create? media or network?

**Reply by NickM-27** [reply]
It needs to be media type

**Reply by DanGarion** [reply]
Hey @NickM-27 I have a question the documentation states (and I've noticed that Frigate saves to '/media/frigate' however when you set up storage in HA they are all at '/mount/XXXX'. I have been pulling my hair trying to figure out why my HA drive keeps getting full and I finally sat down to take a look. How do I get Frigate to use the '/mount/frigate' share that HA creates instead it trying to save to '/media/frigate'?

I can't find a setting in frigate.yml except for where the DB saves.

**Reply by NickM-27** [reply]
@DanGarion you don't, as the docs explain you create the share at /media/frigate

**Reply by DanGarion** [reply]
Nevermind, I just was able to answer my own question with some sleuthing. Both '/mount/frigate' and '/media/frigate' end up pointing to the same location... that is a bit frustrating to find out but I guess that explains it.

**Reply by DanGarion** [reply]
When you use the HA storage tool you cannot tell it to place the share in media instead of mount. When you create a backup network storage it creates the it in mount, when you create a share it also creates it in mount. When you use media in network storage is creates it and the same location is in both media and mount. Just a weird way for HA to do it when I'm used to setting up shares outside of HA.
